server/cli/accounting.go

```diff
@@ -48,37 +48,39 @@ func runGetAcct(cmd *cobra.Command, args []string) {
 		cmd.Usage()
 		return
 	}
-	server.RunGetAcct(Context, args[0])
+	client := client.NewAdminClient(&Context.Context, Context.Addr, client.Accounting)
+	body, err := client.GetYAML(args[0])
+	if err != nil {
```

Comment:
the pattern used elsewhere is:
```
	if err != nil {
		fmt.Fprintf(osStderr, "failed to generate CA certificate: %s\n", err)
		osExit(1)
		return
	}
```
which allows osExit to be overwritten in tests